formats/src/main/scala/flatgraph/formats/graphml/package.scala

```diff
@@ -0,0 +1,36 @@
+package flatgraph.formats
+
+package object graphml {
+  // we could/should make these configurable...
+  val KeyForNodeLabel = "labelV"
+  val KeyForEdgeLabel = "labelE"
+
+  object Type extends Enumeration {
+    val Boolean = Value("boolean")
+    val Int     = Value("int")
+    val Long    = Value("long")
+    val Float   = Value("float")
+    val Double  = Value("double")
+    val String  = Value("string")
+
+    def fromRuntimeClass(clazz: Class[_]): Type.Value = {
```

Comment:
Why this check?
I.e. we could just check
```
clazz match {
case classOf[Boolean] | classOf[java.lang.Boolean] => Type.Boolean
...
}
```
Otherwise you would need to document that it is not a problem that e.g.
```
classOf[java.lang.Object].isAssignableFrom(classOf[java.lang.Boolean])
val res44: Boolean = true
```

Reply:
true that. also old code. will change.

Reply:
doen